samples/tls/logging/README.md

```diff
@@ -37,9 +26,37 @@ my_repository_path/logging/bin/deploy_logging_open.sh
 ```
 ## Notes On Customization Values
 
-Set `MON_TLS_ENABLE=true` in the `$USER_DIR/logging/user.env` file. This variable modifies the deployment of Open Distro for Elasticsearch to be TLS-enabled.
+### TLS 
+
+Specify `TLS_ENABLE=true` in the `user.env` file to use TLS for communications between the ingress object and Kibana. Connections between the ingress object and Elasticsearch always use TLS, regardless of the value of `TLS_ENABLE`.
```

Comment:
Thanks for moving this sentence out...just need to tweak the wording to remove the references to ingress objects (since this flag has an impact, even if ingress is not used).  Perhaps:
_Specify...to require TLS for connections to Kibana.  Connections to Elasticsearch itself (when enabled) always requires TLS, regardless of the value of TLS_ENABLE._

Reply:
Changed as noted